marlowe-dashboard-client/src/Css.purs

```diff
@@ -11,8 +15,33 @@ import Halogen.HTML.Properties (IProp, classes)
 classNames :: forall r i. Array String -> IProp ( class :: String | r ) i
 classNames = classes <<< map ClassName
 
+toggleWhen :: Boolean -> String -> String -> Array String
+toggleWhen condition className1 className2 = if condition then [ className1 ] else [ className2 ]
+
 applyWhen :: Boolean -> String -> Array String
 applyWhen condition className = if condition then [ className ] else []
 
 hideWhen :: Boolean -> Array String
 hideWhen condition = applyWhen condition "hidden"
+
+h2Classes :: Array String
+h2Classes =
+  [ "font-bold"
+  , "text-lg"
+  , "mb-1"
+  ]
+
+buttonClasses :: Array String
+buttonClasses =
+  [ "p-0.5"
+  , "leading-none"
+  , "focus:outline-none"
+  , "disabled:opacity-50"
+  , "disabled:cursor-not-allowed"
+  ]
+
+textInputClasses :: Array String
+textInputClasses =
+  [ "border"
+  , "p-0.5"
+  ]
```

Comment:
should these be moved into the tailwind config to share between projects or should we share this file later? For example, I imagine we will want the same buttons in playground eventually.

Reply:
Not in the tailwind config, but I'm sure we will want some shared code for "components" like these. But we can think how best to handle that later.